kuma/wiki/jinja2/wiki/react_document.html

```diff
@@ -1,5 +1,5 @@
-{% set beta = settings.ENABLE_RESTRICTIONS_BY_HOST and is_beta(request) %}
-{% set untrusted = settings.ENABLE_RESTRICTIONS_BY_HOST and is_untrusted(request) %}
+{% set wiki = False %}
```

Comment:
Perhaps add a code comment here that explains that the reason we're doing this is for the benefit of the jinja includes that might need to know if we're on the wiki.